src/Http/Controllers/Conversations/Messages/MessageController.php

```diff
@@ -32,7 +32,7 @@ public function __invoke(Request $request, $conversationId): JsonResponse
             ])
             ->defaultSort('-created_at')
             ->allowedSorts('created_at', 'updated_at')
-            ->with('user', 'users')
+            ->with('users')
```

Comment:
Could this also commonly be seen as redundant data? How ofter do you want to know who read a message?

Reply:
I think it would make sense in this request to see if you read the message. I updated the query to only return the current authenticated user.